README.md

```diff
@@ -120,9 +120,23 @@ PiwikTracker.shared?.logger = DefaultLogger(minLevel: .error)
 You can also write your own `Logger` and send the logs whereever you want. Just write a new class/struct an let it conform to the `Logger` protocol.
 
 ### Custom User Agent
-You can now use custom user agent of `PiwikTracker`. Add a parameter for custom user agent in the constructor of `PiwikTracker`.
+The `PiwikTracker` will create a default user agent derrived from the WKWebView user agent. This is why you should always instantiate and configure the `PiwikTracker` in the main thread.
```

Comment:
Do we need to expose this implementation detail to the user?

Would it be better if the framework run the instantiation of the `WKWebView` on the main thread in stead of pushing this to the user?

I think this should be fixed in code, not be documentation.